frontend/asset_mapper.rst

```diff
@@ -0,0 +1,1030 @@
+install it via npm:
+
+.. code-block:: terminal
+
+    $ npm install -D dart-sass
+
+Next, create an ``assets/styles/app.scss`` file and write some dazzling CSS:
+
+.. code-block:: scss
+
+    /* assets/styles/app.scss */
+    $primary-color: skyblue;
+
+    body {
+        background: $primary-color;
+    }
+
+Then, run the ``dart-sass`` binary in "watch" mode to build the CSS file to a
+new ``assets/styles/app.css`` path:
+
+.. code-block:: terminal
+
+    $ npx dart-sass assets/styles/app.scss assets/styles/app.css --watch
+
+    # or with the standalone binary:
+    ./sass assets/styles/app.scss assets/styles/app.css --watch
+
+In your template, point directly to the ``styles/app.css`` file (``base.html.twig``
+points to ``styles/app.css`` by default):
+
+.. code-block:: html+twig
+
+    {# templates/base.html.twig #}
+    <link rel="stylesheet" href="{{ asset('styles/app.css') }}">
+
+Done! You can choose to ignore the ``assets/styles/app.css`` file from Git
+or commit it to ease deployment. To prevent the source ``.scss`` files from being
+exposed to the public, see :ref:`exclude_patterns <excluded_patterns>`.
+
+Third-Party Bundles & Custom Asset Paths
+----------------------------------------
+
+All bundles that have a ``Resources/public/`` or ``public/`` directory will
+automatically have that directory added as an "asset path", using the namespace:
+``bundles/<BundleName>``. For example, if you're using `BabdevPagerfantaBundle`_
+if you run the ``debug:asset-map`` command, you'll see an asset whose logical
+path is ``bundles/babdevpagerfanta/css/pagerfanta.css``.
+
+This means you can render these assets in your templates using the
+``asset()`` function:
+
+.. code-block:: html+twig
+
+    <link rel="stylesheet" href="{{ asset('bundles/babdevpagerfanta/css/pagerfanta.css') }}">
+
+Actually, this path - ``bundles/babdevpagerfanta/css/pagerfanta.css`` - already
+works in applications *without* AssetMapper, because the ``assets:install`` command copies
+the assets from bundles into ``public/bundles/``. However, when AssetMapper is
+enabled, the ``pagerfanta.css`` file will automatically  be versioned! It will
+output someting like:
+
+.. code-block:: html+twig
+
+    <link rel="stylesheet" href="/assets/bundles/babdevpagerfanta/css/pagerfanta-ea64fc9c55f8394e696554f8aeb81a8e.css">
+
+Overriding 3rd-Party Assets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to override a 3rd-party asset, you can do that by creating a
+file in your ``assets/`` directory with the same name. For example, if you
+want to override the ``pagerfanta.css`` file, create a file at
+``assets/bundles/babdevpagerfanta/css/pagerfanta.css``. This file will be
+used instead of the original file.
+
+.. note::
+
+    If a bundle renders their *own* assets, but they use a non-default
+    :ref:`asset package <asset-packages>`, then AssetMapper will not be used.
+    This happens, for example, with EasyAdminBundle.
```

Comment:
Like to https://symfony.com/bundles/EasyAdminBundle/current/index.html , just like we do for BabdevPagerfantabundle?